Assets/Scripts/Items/Equipments/FireArmor.cs

```diff
@@ -0,0 +1,35 @@
+namespace Assets.Scripts.Items.Equipments
+{
+    using Enums;
+    using Interfaces;
+    using UnityEngine;
+    using System.Collections;
+    using Enemies;
+    using NoMonoBehaviour;
+
+    [DisallowMultipleComponent]
+
+    public class FireArmor : BaseItem, IEquipment
+    {
+        [SerializeField] private float _changeMaxHealth;
+        [SerializeField] private float _changeRegeneration;
+        [SerializeField] private GameObject FireCircke;
```

Comment:
```suggestion
        [SerializeField] private GameObject _fireCircke;
```
Поля именуются с префиксом _pascalCase